projects/plugins/boost/app/assets/src/js/utils/generate-critical-css.ts

```diff
@@ -210,6 +248,16 @@ async function generateForKeys(
 		}
 	}
 
+	// Track complete Critical CSS generation result.
+	const { progress, percent_complete } = getCurrentStatus();
+	const trackerAttributes: TrackerAttributes = {
+		status: 'success',
+		progress,
```

Comment:
I think at this stage, progress and percent_complete are not yet fully updated as shown in the screenshot below:

<img width="1535" alt="Screen Shot 2021-09-16 at 8 31 25 PM" src="https://user-images.githubusercontent.com/927932/133612886-011a5409-277e-405f-bc03-c975546de542.png">
Maybe also they are irrelevant in the case of success. We assume that the process as completed?

Reply:
We also might want to send a final Track before the `return` at line 246 in case of a show-stopper error?

Reply:
Finally, I am wondering it could be useful to send the provider_keys in here so have more details on the providers

Reply:
A list of provider keys does sound useful. :)

Reply:
What are provider_keys?

Reply:
> What are provider_keys?

When generating Critical CSS, we slice the user's site up into areas which are likely to use the same templates / CSS files. Each slice is identified by a short string called a "provider_key" (for internal / historic reasons).